app/views/npq_separation/admin/users/_application.html.erb

```diff
@@ -0,0 +1,51 @@
+<%- title = "Application #{application_iteration.index + 1}" -%>
+<%- actions = [govuk_link_to("View full application", npq_separation_admin_application_path(application))] -%>
+
+<%=
+  govuk_summary_list(card: { title:, actions: }) do |sl|
+    sl.with_row do |row|
+      row.with_key(text: "Application ID")
+      row.with_value(text: application.id)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "Lead Provider")
+      row.with_value(text: application.lead_provider.name)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "Lead Provider Approval Status")
+      row.with_value(text: application.lead_provider_approval_status)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "NPQ Course")
+      row.with_value(text: application.course.name)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "School URN")
+      row.with_value(text: application.school.urn)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "School UKPRN")
+      row.with_value(text: application.school.ukprn)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "Funded Place?")
```

Comment:
maybe this has been agreed, but should funded place have a question mark in the key? Also we have three values for funded place `nil`, `true` and `false`, funded place only is set on applications from cohort 2024 onwards on approved applications. Otherwise the value is `nil` as in previous cohorts it's not applicable, maybe we should distinguish those in the method below? If this has been already agreed then all good

Reply:
preference is to have that covered if that's ok... eyeing sonar cloud percentages :D